Update SetFollowPedCamThisUpdate (#1137)
citizenfx/natives: CAM/SetFollowPedCamThisUpdate.md

```c
// 0x44A113DD6FFC48D1 0x1425F6AC
BOOL SET_FOLLOW_PED_CAM_THIS_UPDATE(char* camName, int easeTime);
```

Overrides the ped follow camera (not first person camera) with the specified camera. The game loads all camera metadata from `update/update.rpf/x64/data/metadata/cameras.ymt` and `x64a.rpf/data/metadata/cameras.ymt` with the ped follow cameras being of type `camFollowPedCameraMetadata`.

```c
enum ePedCameras {

>     "DEFAULT_FOLLOW_PED_CAMERA",

Reviewer: remove those "", you don't put this in a enum

just 
```c
enum ePedCameras {
DEFAULT_FOLLOW_PED_CAMERA,
}
```
etcc

    FOLLOW_PED_ATTACHED_TO_ROPE_CAMERA,
    FOLLOW_PED_ON_EXILE1_LADDER_CAMERA,
    FOLLOW_PED_SKY_DIVING_CAMERA,
    FOLLOW_PED_SKY_DIVING_FAMILY5_CAMERA,
    NIGHTCLUB_FOLLOW_PED_CAMERA,
    FOLLOW_PED_INTIMIDATION_CAMERA,
    FOLLOW_PED_IN_WATER_CAMERA,
    FOLLOW_PED_PRONE_CAMERA,
    FOLLOW_PED_ON_SEAT_CAMERA,
    FOLLOW_PED_HANGING_UPSIDE_DOWN_CAMERA,
    FOLLOW_PED_ATTACHED_TO_ROPE_CAMERA,
    CUSTOM_TRANSITION_AFTER_WARP_SKY_DIVE_CAMERA,
    FOLLOW_PED_ON_HORSE_CAMERA,
    FOLLOW_PED_ON_LOUNGER_CAMERA,

    // Not yet named camera hashes
    // 0x5DBBFB6E
    // 0xA38DB056
    // 0x16B702A3
    // 0x41D72A2E
}

> ```

Reviewer 2: You could do instead:
```c
enum ePedFollowCameras {
    DEFAULT_FOLLOW_PED_CAMERA,
    FOLLOW_PED_ATTACHED_TO_ROPE_CAMERA,
    FOLLOW_PED_ON_EXILE1_LADDER_CAMERA,
    FOLLOW_PED_SKY_DIVING_CAMERA,
    FOLLOW_PED_SKY_DIVING_FAMILY5_CAMERA,
    NIGHTCLUB_FOLLOW_PED_CAMERA,
    FOLLOW_PED_INTIMIDATION_CAMERA,
    FOLLOW_PED_IN_WATER_CAMERA,
    FOLLOW_PED_PRONE_CAMERA,
    FOLLOW_PED_ON_SEAT_CAMERA,
    FOLLOW_PED_HANGING_UPSIDE_DOWN_CAMERA,
    FOLLOW_PED_ATTACHED_TO_ROPE_CAMERA,
    CUSTOM_TRANSITION_AFTER_WARP_SKY_DIVE_CAMERA,
    FOLLOW_PED_ON_HORSE_CAMERA,
    FOLLOW_PED_ON_LOUNGER_CAMERA,
}
```
Other named camera hashes (not found yet)
```c
// 0x5DBBFB6E
// 0xA38DB056
// 0x16B702A3
// 0x41D72A2E
```

It will make easier for the user to copy/paste the enum with the hashes found.

## Parameters
* **camName**: The name of the camera to use this update

> * **easeTime**: The time (in ms) to ease in the camera

Reviewer 2: ```suggestion
* **easeTime**: The time in milliseconds to ease in the camera
```

Better AmE.

> ## Return value
> 

Reviewer 2: After some tests, it seems to return `false` when not successful.

Code used:
```lua
local followCam = SetFollowPedCamThisUpdate("FOLLOW_PED_ATTACHED_TO_ROPE_CAMERA", 500)
if not followCam then
    print('This cam didn\'t work!')
end
```

```suggestion
## Return value
Return `true` if set camera successfully and `false` otherwise.
```

## Examples
```lua
-- Example with zoomed out camera

> while true do

Reviewer: You could add js example too!

> ## Examples
> ```lua
> -- Example with zoomed out camera
> while true do
>     SetFollowPedCamThisUpdate("FOLLOW_PED_ATTACHED_TO_ROPE_CAMERA", 500)
>     Citizen.Wait(0)
> end
> ```

Reviewer 2: ```suggestion
## Examples
```lua
CreateThread(function()
    while true do
        SetFollowPedCamThisUpdate("FOLLOW_PED_ATTACHED_TO_ROPE_CAMERA", 500) -- Zoomed out the ped camera
        Wait(0)
    end
end)
```
```

| Follow Camera Names                          |
|----------------------------------------------|
| DEFAULT_FOLLOW_PED_CAMERA                    |
| FOLLOW_PED_ATTACHED_TO_ROPE_CAMERA           |
| FOLLOW_PED_ON_EXILE1_LADDER_CAMERA           |
| FOLLOW_PED_SKY_DIVING_CAMERA                 |
| FOLLOW_PED_SKY_DIVING_FAMILY5_CAMERA         |
| NIGHTCLUB_FOLLOW_PED_CAMERA                  |
| FOLLOW_PED_INTIMIDATION_CAMERA               |
| FOLLOW_PED_IN_WATER_CAMERA                   |
| FOLLOW_PED_PRONE_CAMERA                      |
| FOLLOW_PED_ON_SEAT_CAMERA                    |
| FOLLOW_PED_HANGING_UPSIDE_DOWN_CAMERA        |
| FOLLOW_PED_ATTACHED_TO_ROPE_CAMERA           |
| CUSTOM_TRANSITION_AFTER_WARP_SKY_DIVE_CAMERA |
| FOLLOW_PED_ON_HORSE_CAMERA                   |
| FOLLOW_PED_ON_LOUNGER_CAMERA                 |

Other camera hashes (names not found yet)
```c
// 0x5DBBFB6E
// 0xA38DB056
// 0x16B702A3
// 0x41D72A2E
```

## Parameters
* **camName**: The name of the camera to use this update
* **easeTime**: The time in milliseconds to ease in the camera

## Return value

> Return `true` if set camera successfully and `false` otherwise.

Reviewer 3: ```suggestion
Return `true` if the camera is set successfully and `false` otherwise.
```
Just realized this typo

```c
// 0x44A113DD6FFC48D1 0x1425F6AC
BOOL SET_FOLLOW_PED_CAM_THIS_UPDATE(char* camName, int easeTime);
```

Overrides the ped follow camera (not first person camera) with the specified camera. The game loads all camera metadata from `update/update.rpf/x64/data/metadata/cameras.ymt` and `x64a.rpf/data/metadata/cameras.ymt` with the ped follow cameras being of type `camFollowPedCameraMetadata`.

> ```c
> enum ePedFollowCameras {
>     DEFAULT_FOLLOW_PED_CAMERA,
>     FOLLOW_PED_ATTACHED_TO_ROPE_CAMERA,
>     FOLLOW_PED_ON_EXILE1_LADDER_CAMERA,
>     FOLLOW_PED_SKY_DIVING_CAMERA,
>     FOLLOW_PED_SKY_DIVING_FAMILY5_CAMERA,
>     NIGHTCLUB_FOLLOW_PED_CAMERA,
>     FOLLOW_PED_INTIMIDATION_CAMERA,
>     FOLLOW_PED_IN_WATER_CAMERA,
>     FOLLOW_PED_PRONE_CAMERA,
>     FOLLOW_PED_ON_SEAT_CAMERA,
>     FOLLOW_PED_HANGING_UPSIDE_DOWN_CAMERA,
>     FOLLOW_PED_ATTACHED_TO_ROPE_CAMERA,
>     CUSTOM_TRANSITION_AFTER_WARP_SKY_DIVE_CAMERA,
>     FOLLOW_PED_ON_HORSE_CAMERA,
>     FOLLOW_PED_ON_LOUNGER_CAMERA,
> }
> ```

Reviewer 3: ```suggestion
| Follow Camera Names                          |
|----------------------------------------------|
| DEFAULT_FOLLOW_PED_CAMERA                    |
| FOLLOW_PED_ATTACHED_TO_ROPE_CAMERA           |
| FOLLOW_PED_ON_EXILE1_LADDER_CAMERA           |
| FOLLOW_PED_SKY_DIVING_CAMERA                 |
| FOLLOW_PED_SKY_DIVING_FAMILY5_CAMERA         |
| NIGHTCLUB_FOLLOW_PED_CAMERA                  |
| FOLLOW_PED_INTIMIDATION_CAMERA               |
| FOLLOW_PED_IN_WATER_CAMERA                   |
| FOLLOW_PED_PRONE_CAMERA                      |
| FOLLOW_PED_ON_SEAT_CAMERA                    |
| FOLLOW_PED_HANGING_UPSIDE_DOWN_CAMERA        |
| FOLLOW_PED_ATTACHED_TO_ROPE_CAMERA           |
| CUSTOM_TRANSITION_AFTER_WARP_SKY_DIVE_CAMERA |
| FOLLOW_PED_ON_HORSE_CAMERA                   |
| FOLLOW_PED_ON_LOUNGER_CAMERA                 |
```
This is one of the few cases this cannot be an enum, using an enum would mean these are `0, 1, 2, 3`, etc when they're actually strings.

Other camera hashes (names not found yet)
```c
// 0x5DBBFB6E
// 0xA38DB056
// 0x16B702A3
// 0x41D72A2E
```

## Parameters
* **camName**: The name of the camera to use this update
* **easeTime**: The time in milliseconds to ease in the camera

## Return value

> Return `true` if set camera successfully and `false` otherwise.

Reviewer 3: ```suggestion
Return `true` if set camera successfully and `false` otherwise.

```

Not sure if this will mess up anything when rendering and I don't really want to find out :)
